packages/react/src/components/loadingSpinner/LoadingSpinner.module.scss

```diff
@@ -55,8 +55,8 @@
 }
 
 @keyframes spinner-color-animation {
-  0% {}
+  0% { /* empty */ }
   33.3% { border-top-color: var(--spinner-color-stage2) }
   66.6% { border-top-color: var(--spinner-color-stage3) }
-  100% {}
+  100% { /* empty */ }
 }
```

Comment:
Could this "no new line" be also fixed?

Reply:
It's rule `block-no-empty` that it nags about... dunno how to quickly fix without just empty-commenting it

Reply:
I meant there is not a new line in the end of file

Reply:
Ah Ok 😂

Reply:
Hmm... stylelint has deprecated the rule 🤔